From: Ben Hutchings <ben@decadent.org.uk>
Subject: qxl: Disable by default
Forwarded: not-needed

This kernel driver is incompatible with older versions of
xserver-xorg-video-qxl, so do not enable it by default.  It should be
enabled by a modprobe configuration file installed by
xserver-xorg-video-qxl in jessie.

--- a/drivers/gpu/drm/qxl/qxl_drv.c
+++ b/drivers/gpu/drm/qxl/qxl_drv.c
@@ -47,7 +47,7 @@ static DEFINE_PCI_DEVICE_TABLE(pciidlist
 };
 MODULE_DEVICE_TABLE(pci, pciidlist);
 
-static int qxl_modeset = -1;
+static int qxl_modeset = 0;
 int qxl_num_crtc = 4;
 
 MODULE_PARM_DESC(modeset, "Disable/Enable modesetting");
